assets/app/scripts/directives/oscFileInput.js

```diff
@@ -31,6 +41,28 @@ angular.module('openshiftConsole')
           };
           reader.readAsText(file);
         });
+        $('.btn-file').on('click', function() {
+          delete scope.file;
```

Comment:
Is this triggered when the button is disabled (since it's just a span)? What happens if I cancel the dialog?